website/source/docs/secrets/gcp/index.html.md

```diff
@@ -427,3 +459,21 @@ for more details.
 [resource-name-relative]: https://cloud.google.com/apis/design/resource_names#relative_resource_name
 [quotas]: https://cloud.google.com/compute/quotas
 [service-accounts]: https://cloud.google.com/compute/docs/access/service-accounts
+
+
+## Upgrade Guides
```

Comment:
I think this is okay here. I suggest merging the stuff at the top here and changing the note at the top to be something like "If you're using an old version of the plugin, please upgrade. Read more in the upgrade guide (link)"